apps/cbportal/base/cloudbroker__node/methodclass/cloudbroker_node.py

```diff
@@ -133,3 +133,36 @@ def execute_script(self, nid, gid, script):
         )
         if jobinfo["state"] == "ERROR":
             raise exceptions.Error("Failed to execute maintenance script")
+
+    def _apply_node_ipmi_action(self, nid, cmd):
+        node = self._getNode(nid)
+        gid = j.application.whoAmI.gid
```

Comment:
this should come from the node object not the global one from the portal instance.